src/i18n/es.json

```diff
@@ -75,6 +78,9 @@
           "deliveryBegin": "Este envío.",
           "deliveryEnd": "Entrega Est."
         }
+      },
+      "defaultParcelSize": {
+        "label": "Tamaño del paquete por defecto"
```

Comment:
This sounds weird to me. What do you think about "Tamaño por defecto del paquete"?